tests/integration/test_interaction.py

```diff
@@ -657,6 +656,23 @@ def test_do_attempt_feedback_is_updated(self):
         expected_feedback = "\n".join(feedback_lines)
         self.assertEqual(self._get_feedback().text, expected_feedback)
 
+    def test_multiple_misplaced_item_feedback(self):
```

Comment:
@e-kolpakov Could you rename this test to something like `test_per_item_feedback`, to distinguish it from other tests that deal with "global" feedback displayed on submit?

Also, it would probably be worth extending this test to cover scenarios where

* no item was misplaced
* a single item was misplaced